async/runtime.mli

```diff
@@ -35,6 +35,7 @@ val run_request :
      region:string
   -> access_key:string
   -> secret_key:string
+  -> ?token:string
```

Comment:
Could you add documentation for `?token` into the odoc below?

Reply:
Should be added in 5e3e783 :)